stock-app/src/test/java/edu/JasonFlinkstrom/advancedjava/stock/StockAppTest.java

```diff
@@ -0,0 +1,18 @@
+package edu.JasonFlinkstrom.advancedjava.stock;
+
+import org.junit.Test;
+
+/**
+* Test StockApp class using mockito
+* NOTE: does not work yet
+ */
+public class StockAppTest {
+    /**
+     * does not work yet, throws AssertionError
+     */
+//    @Test(expected=NullPointerException.class)

```

Comment:
use system rules to test main method 
http://stefanbirkner.github.io/system-rules/